data/.gitignore

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
@OscarGonzalezLesaga Are you sure that this gitignore is necessary? I have removed it, compiled the project, and nothing is added to git

Reply:
I think that the project get .gitignore of the general folder, so... We can delete files of modules.... Thanks! :)